packages/cbioportal-frontend-commons/src/components/signal/MutationTumorTypeFrequencyTable.tsx

```diff
@@ -0,0 +1,380 @@
+import {
```

Comment:
This was moved from `packages/react-variant-view/src/component/frequencyTable/MutationTumorTypeFrequencyTable.tsx`, right? It is weird that it shows up as a new file.

Reply:
I moved this to cbioportal-commons because now we use it in both cbioportal and signal